src/back/deps/svetit/shared/db/db_table.hpp

```diff
@@ -144,39 +144,50 @@ auto Table<T>::Create(const T& item)
 	static const auto createSql = []() -> storages::postgres::Query {
 		auto names = TableFields();
 		auto idsIndexes = utils::IdsTuple<T>::Get();
-		std::string fields, values, idRet;
+		auto skipFieldsIndexes = utils::NoInsertIdsTuple<T>::Get();
+		std::string fields, values, idsRet, fieldsSkip;
 		std::size_t nameIndex = 0;
 		for (auto&& name : names) {
-			if (utils::TupleContains(idsIndexes, nameIndex++)) {
-				if (idRet.empty())
-					idRet = fmt::format("{},${}", name, nameIndex);
+			if (utils::TupleContains(idsIndexes, nameIndex)) {
+				if (!idsRet.empty())
+					idsRet += ", ";
+				idsRet += name;
+			}
 
-				if (name == "id")
-					continue;
+			if (utils::TupleContains(skipFieldsIndexes, nameIndex)) {
+				if (!fieldsSkip.empty())
+					fieldsSkip += ", ";
+				fieldsSkip += fmt::format("${}", ++nameIndex);
+				continue;
 			}
 
 			if (!fields.empty()) {
 				fields += ", ";
 				values += ", ";
 			}
 			fields += name;
-			values += fmt::format("${}", nameIndex);
+			values += fmt::format("${}", ++nameIndex);
 		}
 
-		if (idRet.empty() && !names.empty())
-			idRet = fmt::format("{},${}", names.front(), 1);
+		if (!idsRet.empty() && !fieldsSkip.empty()) {
+			idsRet += ", ";
+			idsRet += fieldsSkip;
+		}
```

Comment:
Кажется должно быть так:
```suggestion
		if (!fieldsSkip.empty()) {
			if (!idsRet.empty())
				idsRet += ", ";
			idsRet += fieldsSkip;
		}
```